src/wp-includes/formatting.php

```diff
@@ -2035,7 +2035,15 @@ function sanitize_file_name( $filename ) {
 	}
 
 	if ( $utf8_pcre ) {
-		$filename = preg_replace( "#\x{00a0}#siu", ' ', $filename );
+		/**
+		 * Replace all whitespace characters with a basic space (U+0020).
+		 *
+		 * Characters in the White_Space category are listed with “Zs” in
+		 * their entry in the UnicodeData file maintained at the linked URL.
```

Comment:
[I think the correct name for this property is "space separator," not "white space".
](https://www.unicode.org/reports/tr44/#General_Category_Values)

```suggestion
		 * Replace all whitespace characters with a basic space (U+0020).
		 *
		 * Characters in the  “Space_Separator” category are listed with “Zs” in
		 * their entry in the UnicodeData file maintained at the linked URL.
```

> `Zl`  Line separator 
> `Zp`  Paragraph separator 
> `Zs`  Space separator

Reply:
interesting. [this document](https://www.unicode.org/reports/tr44/) is confusing to me. I think the _category_ of `Space_Separator` (`Zs`) is meant to include characters with the binary `White_Space` property. I’m not entirely sure that all characters with the `White_Space` property are in the `Space_Separator` category.

since this is directly referencing `Zs` in the PCRE pattern, I think your suggestion is best.

Reply:
I typed with the wording; would appreciate your thoughts

Reply:
I also find the document confusing. I'm reasonably confident that the `Zs` category is the correct choice, but I'm less confident about the language we're using to describe it.

I think the modified comment as you've authored it is good 👍  it explains the intent and it links to the relevant documentation for readers to investigate further.

Reply:
after @adamziel raised his own similar questions I expanded the comment with more links